x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/system/generic_file_details.tsx

```diff
@@ -157,7 +157,13 @@ export const SystemGenericFileLine = React.memo<Props>(
             processExecutable={processExecutable}
           />
         </TokensFlexItem>
-        <Args args={args} contextId={contextId} eventId={id} processTitle={processTitle} />
+        <Args
+          args={args}
+          contextId={contextId}
+          eventId={id}
+          processTitle={processTitle}
+          isDraggable={isDraggable}
```

Comment:
This fix a bug where Args where draggable even when isDeggable is falsy

Reply:
good catch here